fuel/transformers/image.py

```diff
@@ -1144,10 +1144,17 @@ def transform_source_example(self, example, source_name):
 
     def _example_transform(self, example, rotation_angle):
         dt = example.dtype
-        im = Image.fromarray(example.transpose(1, 2, 0))
-        example = numpy.array(im.rotate(rotation_angle,
-                                        resample=self.resample)).astype(dt)
-        return example.transpose(2, 0, 1)
+        if dt == 'float32' or dt == 'float64':
```

Comment:
I think the best way to check if the dtype is float is:

if np.issubdtype(dt, np.float):